test/specs/anInitialSuite.e2e.ts

```diff
@@ -107,68 +102,53 @@ describe('An Initial Suite', async () => {
-      await utilities.runCommandFromCommandPrompt(
-        workbench,
-        'Developer: Show Running Extensions',
-        2
-      );
-
-      // Close panel so the visualforce extension can be seen
-      await utilities.runCommandFromCommandPrompt(
-        workbench,
-        'View: Close Panel',
-        1
-      );
-
-      let matchesFound = 0;
-      const extensionNameDivs = await $$('div.name');
-      for (const extensionNameDiv of extensionNameDivs) {
-        const text = await extensionNameDiv.getText();
-
-        if (text.startsWith('salesforce.salesforcedx-vscode-')) {
-          matchesFound++;
-          utilities.log(`AnInitialSuite - extension ${text} is loaded`);
-        }
+  step('Verify our extensions are loaded after creating an SFDX project', async () => {
+    const workbench = await browser.getWorkbench();
+    await utilities.runCommandFromCommandPrompt(workbench, 'Developer: Show Running Extensions', 2);
+
+    // Close panel so the visualforce extension can be seen
+    await utilities.runCommandFromCommandPrompt(workbench, 'View: Close Panel', 1);
+
+    let matchesFound = 0;
+    const extensionNameDivs = await $$('div.name');
+    for (const extensionNameDiv of extensionNameDivs) {
+      const text = await extensionNameDiv.getText();
+
+      if (text.startsWith('salesforce.salesforcedx-vscode-')) {
+        matchesFound++;
+        utilities.log(`AnInitialSuite - extension ${text} is loaded`);
       }
-      expect(matchesFound).toBe(7);
     }
-  );
-
-  step(
-    'Verify that SFDX commands are present after an SFDX project has been created',
-    async () => {
-      const workbench = await browser.getWorkbench();
-      const prompt = await utilities.openCommandPromptWithCommand(
-        workbench,
-        'SFDX:'
-      );
-      const quickPicks = await prompt.getQuickPicks();
-      const commands: string[] = [];
-      for (const quickPick of quickPicks) {
-        commands.push(await quickPick.getLabel());
-      }
 
-      // Look for the first few SFDX commands.
-      expect(commands).toContain('SFDX: Add Tests to Apex Test Suite');
-      expect(commands).toContain('SFDX: Authorize a Dev Hub');
-      expect(commands).toContain('SFDX: Authorize an Org');
-      expect(commands).toContain('SFDX: Authorize an Org using Session ID');
-      expect(commands).toContain('SFDX: Cancel Active Command');
-      expect(commands).toContain('SFDX: Configure Apex Debug Exceptions');
-      expect(commands).toContain('SFDX: Create a Default Scratch Org...');
-      expect(commands).toContain('SFDX: Create Apex Class');
-      expect(commands).toContain('SFDX: Create Apex Test Suite');
-      expect(commands).toContain('SFDX: Create Apex Trigger');
-      // There are more, but just look for the first few commands.
-
-      // Escape out of the pick list.
-      await prompt.cancel();
+    // 7 are in the list, but only 6 are visible.
+    expect(matchesFound).toBeGreaterThanOrEqual(6);
```

Comment:
On my machine's screen (Mac laptop), sometimes 6 would be shown, and sometimes 7 would be shown... it all depends on the order the other extensions finished loading.